xml/System/DateTime.xml

```diff
@@ -46,351 +46,353 @@
+[!code-vb[System.DateTime.Persistence#2](~/samples/snippets/visualbasic/System.DateTime/Persistence.vb#2)]  
+
+#### Persisting values as integers  
+
+You can persist it as an <xref:System.Int64> value that represents a number of ticks. In this case, you don't have to consider the culture of the systems the <xref:System.DateTime> values are persisted and restored on.  
   
- To persist a <xref:System.DateTime> value as an integer:  
+To persist a <xref:System.DateTime> value as an integer:  
   
 -   If the <xref:System.DateTime> values represent single moments in time, convert them to UTC by calling the <xref:System.DateTime.ToUniversalTime%2A> method.  
-  
 -   Retrieve the number of ticks represented by the <xref:System.DateTime> value from its <xref:System.DateTime.Ticks%2A> property.  
   
- To restore a <xref:System.DateTime> value that has been persisted as an integer:  
+To restore a <xref:System.DateTime> value that has been persisted as an integer:  
   
 1.  Instantiate a new <xref:System.DateTime> object by passing the <xref:System.Int64> value to the <xref:System.DateTime.%23ctor%28System.Int64%29> constructor.  
+1.  If the <xref:System.DateTime> value represents a single moment in time, convert it from UTC to the local time by calling the <xref:System.DateTime.ToLocalTime%2A> method.  
+  
+The following example persists an array of <xref:System.DateTime> values as integers on a system in the U.S. Pacific Time zone. It restores it on a system in the UTC zone. The file that contains the integers includes an <xref:System.Int32> value that indicates the total number of <xref:System.Int64> values that immediately follow it.  
+  
+[!code-csharp[System.DateTime.Persistence#3](~/samples/snippets/csharp/System.DateTime/Persistence.cs#3)]
+[!code-vb[System.DateTime.Persistence#3](~/samples/snippets/visualbasic/System.DateTime/Persistence.vb#3)]  
+
+#### Serializing DateTime values  
+
+You can persist <xref:System.DateTime> values through serialization to a stream or file, and then restore them through deserialization. <xref:System.DateTime> data is serialized in some specified object format. The objects are restored when they are deserialized. A formatter or serializer, such as <xref:System.Xml.Serialization.XmlSerializer> or <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter>, handles the process of serialization and deserialization. For more information about serialization and the types of serialization supported by the .NET Framework, see [Serialization](http://msdn.microsoft.com/library/4d1111c0-9447-4231-a997-96a2b74b3453).  
+  
+The following example uses the <xref:System.Xml.Serialization.XmlSerializer> class to serialize and deserialize <xref:System.DateTime> values. The values represent all leap year days in the twenty-first century. The output represents the result if the example is run on a system whose current culture is English (G
```

Comment:
Why is this information useful?

Reply:
@rpetrusha I had the same thought as I was updating this. Do you remember? Should it be deleted?

Reply:
@BillWagner and @svick, I'm sorry, I don't quite follow. What about line 381, what's showing incorrectly, and what should be deleted? (I'm probably missing something glaringly obvious.)

Reply:
@rpetrusha Line 381 is:

> ### Version considerations  
> 
> Starting with the .NET Framework 2.0, the `System.DateTime` structure contains a 64-bit field composed of a private Kind field concatenated with the Ticks field. The Kind field is a 2-bit field that indicates whether the `System.DateTime` structure represents a local time, a Coordinated Universal Time (UTC), or the time in an unspecified time zone. The Kind field is used when performing time conversions between time zones, but not for time comparisons or arithmetic. The value of the Kind field can be obtained with the `DateTime.Kind` property.

I don't see why would this information be useful now.

I commented about this on line 381, but for some reason, my comment is incorrectly showing here.

Reply:
Thanks for the clarification, @svick. I agree, @BillWagner, that it should be removed.